### PR TITLE
multiple CBS URLs in configs

### PR DESCRIPTION
**Pull request by jamiltz**
changes:

- callout on whatsnew page.
- document `server` prop in SG config.
- document `server` prop for data and channel buckets in SGA config.

relates to https://github.com/couchbase/sync_gateway/issues/1019, https://github.com/couchbase/sync_gateway/issues/942
fix https://github.com/couchbaselabs/couchbase-mobile-portal/issues/791

### CODE REVIEW

File: configs/15/sg-accel.yaml [hunk 1]
```diff
@@ -52,7 +90,23 @@ properties:
...
+                      - `couchbase://host1:8091,host2,`
+                      - `couchbase://host1:18091,host2:8091`
+                      - `http://host1:8091`
+                      - `http://host1,host2:8091`
+                      - `http://foo:bar@host1:8091`
+                      
+                    On startup, Sync Gateway Accelerator will try each hostname that is provided until it is able to connect successfully.
+                    
+                    If the connection to Couchbase Server is lost during normal operations, Sync Gateway Accelerator will automatically re-connect to another node in the cluster.
                 bucket:
```
Comment (jamiltz) [inline configs/15/sg-accel.yaml]: @tleyden not totally related to this PR, could you double check that the `server` key is valid in 3 different places in the SGA config codebase?

Also the descriptions are slightly different for each one (`channel_index.server` doesn't mention walrus for example). There might be more to write on the specific behaviour of each one, I'm not sure.

**Comment by tleyden** [comment]
Hold off on this change since `couchbase://host1:8091` is not actually valid for now.  @adamcfraser is working on a fix for https://github.com/couchbase/sync_gateway/issues/2810

**Comment by tleyden** [comment]
@adamcfraser any chance you can take a look at these docs changes in light of recent fixes?

**Comment by raghusarangapani** [comment]
This ticket does not say that https:// is not supported and that customers should use couchbases:// instead @djpongh @adamcfraser

**Comment by jamiltz** [comment]
@adamcfraser re-assigned PR to you.

**Pull request merged**